rustfmt.toml

```diff
@@ -1,2 +1,10 @@
-tab_spaces = 2
 edition = "2021"
+tab_spaces = 2
```

Comment:
I like this, but I want to improve the code styles after https://github.com/napi-rs/napi-rs/pull/1074